### PR TITLE
if next image src different from current update the displayed image

### PR DESCRIPTION
**Pull request by conorhastings**
closes #4

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,26 +1,37 @@
-import React from "react";
+import React, { Component, PropTypes } from "react";
```
Comment (rickyvetter) [inline src/index.js]: Does this still build correctly without having React.createElement present?

Comment (conorhastings) [inline src/index.js]: yeah

Comment (rickyvetter) [inline src/index.js]: That's cool - do you know why?

Comment (conorhastings) [inline src/index.js]: you're still passing in all of React, just destructuring those so you don't have to refer to React.Component, React.PropTypes, I assume it wouldn't work if it was just

```js
import { Component, PropTypes } from "react";
```

Comment (rickyvetter) [inline src/index.js]: Oh wow. Totally misread the line and thought that you had removed the React in addition to adding the destructuring. Coolio :+1:

**Pull request merged**